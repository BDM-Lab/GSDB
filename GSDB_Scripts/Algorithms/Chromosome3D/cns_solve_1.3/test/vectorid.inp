remarks IDENTIFY_TEST.INP
remarks test case for IDENT command

topology
   !
   mass A   1.0
   mass B   2.0
   mass C   3.0
   mass D   4.0
   mass E   5.0
   mass F   6.0
   mass G   7.0
   mass H   8.0
   mass I   9.0
   !
   residue test 
     atom A  type A  charge 10.0 end
     atom B  type B  charge 20.0 end
     atom C  type C  charge 30.0 end
     atom D  type D  charge 40.0 end
     atom E  type E  charge 50.0 end
     atom F  type F  charge 60.0 end
     atom G  type G  charge 70.0 end
     atom H  type H  charge 80.0 end
     atom I  type I  charge 90.0 end
   end
   !
end !topology

segment
   name=test
   molecule name=test number=1 end
end

   iden ( b )     ( name a or name b or name c )
   show element ( b ) ( all )

stop 
