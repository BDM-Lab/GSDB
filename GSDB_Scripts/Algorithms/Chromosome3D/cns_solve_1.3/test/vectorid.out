          ============================================================
          |                                                          |
          |            Crystallography & NMR System (CNS)            |
          |                         CNSsolve                         |
          |                                                          |
          ============================================================
           Version: 1.3
           Status: General release
          ============================================================
           Written by: A.T.Brunger, P.D.Adams, G.M.Clore, W.L.DeLano,
                       P.Gros, R.W.Grosse-Kunstleve,J.-S.Jiang,J.M.Krahn,
                       J.Kuszewski, M.Nilges, N.S.Pannu, R.J.Read,
                       L.M.Rice, G.F.Schroeder, T.Simonson, G.L.Warren.
           Copyright (c) 1997-2010 Yale University
          ============================================================
           Running on machine: lewis4-r710-login-node223 (x86_64/Linux,64-bit)
                               with 16 threads
           Program started by: oeow39
           Program started at: 16:08:22 on 05-Jul-2018
          ============================================================

 FFT3C: Using FFTPACK4.1

 CNSsolve>remarks IDENTIFY_TEST.INP 
 CNSsolve>remarks test case for IDENT command 
 CNSsolve> 
 CNSsolve>topology 
 RTFRDR>   ! 
 RTFRDR>   mass A   1.0 
 RTFRDR>   mass B   2.0 
 RTFRDR>   mass C   3.0 
 RTFRDR>   mass D   4.0 
 RTFRDR>   mass E   5.0 
 RTFRDR>   mass F   6.0 
 RTFRDR>   mass G   7.0 
 RTFRDR>   mass H   8.0 
 RTFRDR>   mass I   9.0 
 RTFRDR>   ! 
 RTFRDR>   residue test 
 RESIDUE>     atom A  type A  charge 10.0 end 
 RESIDUE>     atom B  type B  charge 20.0 end 
 RESIDUE>     atom C  type C  charge 30.0 end 
 RESIDUE>     atom D  type D  charge 40.0 end 
 RESIDUE>     atom E  type E  charge 50.0 end 
 RESIDUE>     atom F  type F  charge 60.0 end 
 RESIDUE>     atom G  type G  charge 70.0 end 
 RESIDUE>     atom H  type H  charge 80.0 end 
 RESIDUE>     atom I  type I  charge 90.0 end 
 RESIDUE>   end 
 RTFRDR>   ! 
 RTFRDR>end !topology 
 CNSsolve> 
 CNSsolve>segment 
 SEGMENT>   name=test 
 SEGMENT>   molecule name=test number=1 end 
 SEGMENT>end 
 Status of internal molecular topology database:
 -> NATOM=          9(MAXA=     1000000)  NBOND=          0(MAXB=     1000000)
 -> NTHETA=         0(MAXT=     2000000)  NGRP=           1(MAXGRP=   1000000)
 -> NPHI=           0(MAXP=     2000000)  NIMPHI=         0(MAXIMP=   1000000)
 -> NNB=            0(MAXNB=    1000000) 
 CNSsolve> 
 CNSsolve>   iden ( b )     ( name a or name b or name c ) 
 SELRPN:      3 atoms have been selected out of      9
 CNSsolve>   show element ( b ) ( all ) 
 SELRPN:      9 atoms have been selected out of      9
 ( TEST TEST 1    A    )    1.0000    
 ( TEST TEST 1    B    )    2.0000    
 ( TEST TEST 1    C    )    3.0000    
 ( TEST TEST 1    D    )    0.0000    
 ( TEST TEST 1    E    )    0.0000    
 ( TEST TEST 1    F    )    0.0000    
 ( TEST TEST 1    G    )    0.0000    
 ( TEST TEST 1    H    )    0.0000    
 ( TEST TEST 1    I    )    0.0000    
 CNSsolve> 
 CNSsolve>stop 
          ============================================================
           Maximum dynamic memory allocation:     8201488 bytes
           Maximum dynamic memory overhead:           208 bytes
           Program started at: 16:08:22 on 05-Jul-2018
           Program stopped at: 16:08:22 on 05-Jul-2018
           CPU time used:       0.0054 seconds
          ============================================================
